client/pages/view-message.jsx

```diff
@@ -131,31 +135,41 @@ function IntroSlide(props) {
   );
 }
 
-function ContentSlide(props) {
-  if (props.currentSlide !== props.slideIndex) {
-    return null;
-  } else {
-    let redirect;
-    if (props.isRecipient) {
-      redirect = '/';
+class ContentSlide extends React.Component {
+  // eslint-disable-next-line no-useless-constructor
+  constructor(props) {
+    super(props);
+  }
```

Comment:
You can just delete the whole constructor if all you're doing is `super(props)`. JavaScript takes care of calling `super()` for you if you leave out the constructor.